presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -459,6 +467,12 @@ class SystemConfig : public ConfigBase {
   bool includeNodeInSpillPath() const;
 
   int32_t oldTaskCleanUpMs() const;
+
+  bool internalCommunicationJwtEnabled() const;
```

Comment:
Do we have both `internal-communication.jwt.enabled` and `internal-communication.shared-secret` on the Java side?
`internal-communication.shared-secret` is sufficient to determine enabling JWT.
That would clean up the semantics.
```
bool internalCommunicationJwtEnabled() const {
  return !(optionalProperty(kInternalCommunicationSharedSecret).value().empty());
}
```

Reply:
Yes, on the Java side both knobs are present.